airflow/providers/docker/operators/docker.py

```diff
@@ -451,11 +451,11 @@ def copy_from_docker(container_id, src):
                 # 0 byte file, it can't be anything else than None
                 return None
             # no need to port to a file since we intend to deserialize
-            file_standin = BytesIO(b"".join(archived_result))
-            tar = tarfile.open(fileobj=file_standin)
-            file = tar.extractfile(stat["name"])
-            lib = getattr(self, "pickling_library", pickle)
-            return lib.loads(file.read())
+            with BytesIO(b"".join(archived_result)) as f:
+                tar = tarfile.open(fileobj=f)
+                file = tar.extractfile(stat["name"])
+                lib = getattr(self, "pickling_library", pickle)
+                return lib.loads(file.read())
```

Comment:
```suggestion
                return lib.load(file)
```

I wonder if this is possible

Reply:
Let me try it.